packages/distributor/solana/utils.ts

```diff
@@ -1,9 +1,13 @@
+import { TransferFeeConfig } from "@solana/spl-token";
 import { SignerWalletAdapter } from "@solana/wallet-adapter-base";
 import { Connection, Keypair, PublicKey, Transaction, VersionedTransaction } from "@solana/web3.js";
 import { ContractError } from "@streamflow/common";
 import { ConfirmationParams, signAndExecuteTransaction, ThrottleParams } from "@streamflow/common/solana";
 
 import { fromTxError } from "./generated/errors";
+import { ONE_IN_BASIS_POINTS } from "./constants";
+
+export const divCeilN = (n: bigint, d: bigint): bigint => n / d + (n % d ? BigInt(1) : BigInt(0));
```

Comment:
Since it's Friday, BigInt isn't as big as BN.
Also, BN has all of these operations.

I'll merge this and migrate it with staging so Milan can test it during the weekend, but let's fix this with BN asap :)